content/html/concepts/attributes/terms/class/class.md

```diff
@@ -73,15 +65,21 @@ Below is some markup with the classes `header-1` and `P1` and `P2`. Elements wit
 </html>
 ```
 
+The above code will output in the browser somewhat like this:
+
+![Browser window with different text blocks formatted according to their class attribute](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-classes-example-1.jpg)
+
 ## For JavaScript
 
 The `class` attribute is used to perform [JavaScript](https://www.codecademy.com/resources/docs/javascript) operations on HTML elements. Through the Document Object Model ("DOM") there are various JavaScript methods built for selecting elements via their class.
 
 Assuming the markup from the last example is linked to a JavaScript file:
 
 ```js
-const innerDiv = document.getElementsByClassName('header-1');
-console.log(innerDiv);
+const innerDiv = document.getElementsByClassName('header-1')[0];
+console.log(innerDiv.textContent);
 ```
 
-The code above gets an element with a class property and prints it to the console.
+The code above selects the first element of a given class, and prints its content to the console.
+
+![Javascript console output showing the contents of the header text object](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-classes-example-2.jpg)
```

Comment:
We shouldn't use images for console output. We use a ` ```shell` block instead. Like this:

````
The example above produces the following output:

```shell
Output of example
```
````

Reply:
> We shouldn't use images for console output. We use a `` ```shell `` block instead. Like this:
> 
> ```
> The example above produces the following output:
> 
> ```shell
> Output of example
> ```
> ```

@SSwiniarski very right!! I was motivated on adding images, but for a console output this doesn't apply.

And regarding to my description of what that code does, do you think it's too verbose and should be more generic as you wrote, or I can leave as it is?